src/profiler/wrapper.h

```diff
@@ -0,0 +1 @@
+#include "cudaProfiler.h"
```

Comment:
Could always just add this header to the driver wrapper.h if its needed

Reply:
Ok will do that.

Reply:
Is it ok if I use `allowlist` instead of `whitelist` ? (bindgen version)

Reply:
Also I don't have cuda 11.3 locally but 11.8, any counterindication to use that ? (It does create a lot of unecessary changes)

Reply:
Yes I think should be fine for both (I was using old bindgen which they made big updates too).